### PR TITLE
fix: Adding specific quantity of a part as installed item(s

### LINKED ISSUE #5398: Adding specific quantity of a part as installed item(s)
### Discussed in https://github.com/inventree/InvenTree/discussions/5397

<div type='discussions-op-text'>

<sup>Originally posted by **palagap** August  4, 2023</sup>
Hi there, 

We noticed an issue when trying to install a specific quantity of a part to a build. We’ve replicated this using the InvenTree demo:

1. We start with an item, such as the Blue Chair.

Screenshot 1

2. We attempt to install a Stock Item (e.g. Screws).

Screenshot 2

3. When selecting the stock item, though, we are only given the option to select the full quantity of screws, rather than being able to enter the desired quantity.

Screenshot 3

4. For instance, the BOM shows that only 5 screws are needed.

Screenshot 6

5. However, we have to install all 1300 of them in this case.

Screenshot 5

Is there a way to avoid this issue and only install the correct number of screws by manually entering the desired number of screws to install? Thanks.

</div>

### PR DESCRIPTION
**Pull request by UlugbekErkinov**
fixed: Adding specific quantity of a part as installed item(s

Closes #5398

### CODE REVIEW

**Review by matmair — COMMENTED** [review]
Thank you for the PR @UlugbekErkinov - I will do a short test in a few hours and then approve if applicable.

**Comment by UlugbekErkinov** [comment]
Is that should be manually enter the desired quantity of a stock item to install instead of being forced to install the entire available quantity?

File: InvenTree/stock/serializers.py [hunk 1]
```diff
@@ -490,24 +490,43 @@ class InstallStockItemSerializer(serializers.Serializer):
         label=_('Stock Item'),
         help_text=_('Select stock item to install'),
     )
+    
+    quantity = serializers.IntegerField(
+        min_value=1,  
+        required=True,
```
[the remark this answers is not in the document]
Comment (matmair) [reply InvenTree/stock/serializers.py:496]: yes, that looks good

**Comment by matmair** [comment]
@UlugbekErkinov please address the style errors, we should be good after that.

**Comment by SchrodingersGat** [comment]
@UlugbekErkinov you will need to change the front-end code too, to add this field to the form:

https://github.com/inventree/InvenTree/blob/06eb948528e945b41ea3ceb7ae75709eb1d639e0/InvenTree/templates/js/translated/stock.js#L3141

Add the new `quantity` field to the set of fields:

```json
},
quantity: {},
```

**Comment by SchrodingersGat** [comment]
@UlugbekErkinov have you had a change to have a further look into this?

**Comment by SchrodingersGat** [comment]
Hey @UlugbekErkinov do you have a chance to look at this again soon? I'd like to be able to include this in our upcoming release

**Comment by SchrodingersGat** [comment]
Replaced by https://github.com/inventree/InvenTree/pull/5872

**Pull request closed without merging**